web/components/templates/prompts/experiments/id/experimentIdPage.tsx

```diff
@@ -58,6 +58,22 @@ const ExperimentIdPage = (props: PromptIdPageProps) => {
     ? Object.keys(runs?.[0]?.inputs).map((key) => key)
     : [];
 
+  const renderScoreValue = (score: {
+    value: string | number;
+    valueType: string;
+  }) => {
+    if (score.valueType === "boolean") {
+      return score.value === 1 ? "true" : "false";
+    }
+    if (score.valueType === "string") {
+      return score.value;
+    }
+    if (score.valueType === "number") {
+      return (score.value as number).toFixed(4);
```

Comment:
do we want it to be fixed to 4? maybe we use the `formatNumbers` function. 

it checks to see how long a number is and fixes it at 6 if and only if its longer than 6